src/vs/workbench/workbench.web.main.ts

```diff
@@ -109,6 +110,7 @@ registerSingleton(ITimerService, TimerService, InstantiationType.Delayed);
 registerSingleton(ICustomEndpointTelemetryService, NullEndpointTelemetryService, InstantiationType.Delayed);
 registerSingleton(IDiagnosticsService, NullDiagnosticsService, InstantiationType.Delayed);
 registerSingleton(ILanguagePackService, WebLanguagePacksService, InstantiationType.Delayed);
+registerSingleton(IIssueMainService, IssueFormService, InstantiationType.Delayed);
```

Comment:
This should happen from `vs/workbench/contrib/issue/browser/issue.contribution` and should not be named `XyMainService`

Reply:
move to contrib, but it still is implementing the `IIssueMainService` interface (which is used in electron as well).